lightstep/sdk/metric/exporters/common/common.go

```diff
@@ -0,0 +1,113 @@
+package common
```

Comment:
Thoughts on the placement of this package? Should I maybe move to sdk/metric/internal/common?